docs/src/microphysics/saturation_adjustment.md

```diff
@@ -7,13 +7,13 @@ Mixed-phase saturation adjustment is described by [Pressel2015](@citet).
 ## Moist static energy and total moisture mass fraction
 
 The saturation adjustment solver (specific to our anelastic formulation) takes four inputs:
-    * moist static energy ``e``
-    * total moisture mass fraction ``qᵗ``
-    * height ``z``
-    * reference pressure ``pᵣ``
+* moist static energy ``e``,
+* total moisture mass fraction ``qᵗ``,
+* height ``z``, and
+* reference pressure ``pᵣ``.
 
 Note that moist static energy density ``ρᵣ e`` and moisture density ``ρᵣ qᵗ``
-are prognostic variables for `Breeze.AtmosphereModel` when using `AnelasticFormulation`,
+are prognostic variables for [`AtmosphereModel`](@ref) when using [`AnelasticFormulation`](@ref),
```

Comment:
out of curiosity, is there other formulations available? no, right?

Reply:
not yet. we plan to implement a fully compressible formulation though.